apps/thehive/README.md

```diff
@@ -0,0 +1,5 @@
+# Vast - TheHive integration
```

Comment:
Fixed

Reply:
I unresolved as this does not appear to be done. Did you forget to push?

Reply:
I mistakenly resolved this. Yet, the integration will have other implementations. I thought it would be better for user to have an introduction document if, for any reason, they end up on this folder. 

We can list and document all different implementations here. All seperate implementations (folders) would have their detailed documentation in their folder as well. Let me know if this works or should I simply remove this documentation here.

Reply:
@dominiklohmann  in the last conversation we had in Slack huddle, you mentioned it would be better if on each folder level we have a documentation. Could you provide your latest thoughts on this? I do vote for having this document, though if its required we can change the format a bit.

Reply:
I think this was the point of this whole thread: there _should_ be an `apps/README.md` file.

Reply:
It seems that I was referring to thehive section and not on /apps level. I added readme.md file there as well.
https://github.com/tenzir/vast/blob/story/sc-36270/thehive-analyzer/apps/README.md